src/cashlink.ts

```diff
@@ -37,5 +38,6 @@ startSentry(Vue);
 
 const app = new Vue({
     store,
+    i18n,
```

Comment:
oh alright, thanks for the information. I'll put it back.

Reply:
But since we can't really use `await` globally, would you prefer to do :
```
loadLanguage(detectLanguage()).then(() => {
    const app = new Vue({
        store,
        i18n,
        render: (h) => h(App),
    }).$mount('#app');
});
```
or 
```
(async () => {
    await loadLanguage(detectLanguage());

    const app = new Vue({
        store,
        i18n,
        render: (h) => h(App),
    }).$mount('#app');
})();
```
?

Reply:
The way you did it is fine.